src/components/ComboBox/ComboBox.tsx

```
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Combobox as ComboboxPrimitive } from '@base-ui/react';
+import { ComboboxTrigger } from './ComboBoxTrigger.tsx';
```

ESLint

[error] Missed spacing between "@base-ui/react" and "./ComboBoxTrigger.tsx".

(perfectionist/sort-imports)